### DISCUSSION TITLE
pnpm why/list don't work in monorepo root

### DISCUSSION THREAD

**Discussion by stkevintan** [pnpm/pnpm#3692] [Q&A] [answered]
Hi, guys
I am working on a monorepo with pnpm, since the packages was too many and most of dependencies are hoist to the root. so I just want to check why some packages were installed, and installed to which package. so I run the `pnpm why <package>` on the root dir. but nothing found. I figure out the fact that I need check the packages one by one which actually is painful.

So, I just wonder why I cannot check the dependencies on the root dir. the lock file,  node_modules are all present on the root

**Comment by zkochan** [comment, marked as answer]
Try `pnpm why -r <package>`

**Reply by stkevintan** [reply]
Got it ,thanks
